src/server/stat_job.c

```diff
@@ -268,23 +268,16 @@ status_job(job *pjob, struct batch_request *preq, svrattrl *pal, pbs_list_head *
 			oldtime = get_jattr_long(pjob, JOB_ATR_eligible_time);
 			set_jattr_l_slim(pjob, JOB_ATR_eligible_time,
 					time_now - get_jattr_long(pjob, JOB_ATR_sample_starttime), INCR);
-
-			/* Note: ATR_VFLAG_MODCACHE must be set because of svr_cached() does */
-			/* 	 not correctly check ATR_VFLAG_SET */
 		}
 	} else {
-		/* eligible_time_enable is off so,				       */
-		/* clear set flag so that eligible_time and accrue type dont show */
-		old_elig_flags = pjob->ji_wattr[(int)JOB_ATR_eligible_time].at_flags;
-		mark_jattr_not_set(pjob, JOB_ATR_eligible_time);
-		pjob->ji_wattr[(int)JOB_ATR_eligible_time].at_flags |= ATR_MOD_MCACHE;
-
-		old_atyp_flags = pjob->ji_wattr[(int)JOB_ATR_accrue_type].at_flags;
-		mark_jattr_not_set(pjob, JOB_ATR_accrue_type);
-		pjob->ji_wattr[(int)JOB_ATR_accrue_type].at_flags |= ATR_MOD_MCACHE;
-
-		/* Note: ATR_VFLAG_MODCACHE must be set because of svr_cached() does */
-		/*		not correctly check ATR_VFLAG_SET */
+		attribute *attr = get_jattr(pjob, JOB_ATR_eligible_time);
+		/* eligible_time_enable is off so, clear set flag so that eligible_time and accrue type dont show */
+		old_elig_flags = attr->at_flags;
+		ATR_UNSET(attr);
+
+		attr = get_jattr(pjob, JOB_ATR_accrue_type);
+		old_atyp_flags = attr->at_flags;
+		ATR_UNSET(attr);
```

Comment:
Use job level function mark_jattr_not_set() instead and get flags as get_jattr(pjob, JOB_ATR_accrue_type)->flags and get_jattr(pjob, JOB_ATR_eligible_time)->flags